tests/scripts/create-dev-cluster.sh

```diff
@@ -56,21 +70,31 @@ show_info() {
     echo " "
     echo " *** To start using your rook cluster please set the following alias: "
     echo " "
-    echo "    > " alias kubectl=\"$KUBECTL\"
+    echo "    >  alias kubectl=\"$KUBECTL\" "
 }
 
-check_minikube_exists() {
-    if minikube profile list | grep -q $ROOK_PROFILE_NAME; then
-        echo "A minikube profile '$ROOK_PROFILE_NAME' already exists, please use -f to force the cluster creation."
-	exit 1
+check_minikube_profile_exists() {
+    
+	minikube profile list | grep -q $ROOK_PROFILE_NAME
+	local retcode=$?
+	if [ $retcode -eq 0 ]; then
+        echo "A minikube profile '$ROOK_PROFILE_NAME' already exists."
+	echo "Using the existing minikube environment."
+	echo " please use -f to force the minikube  cluster creation."
+	return  1
+    else
+	echo " A minikube environment does not exist yet."
+	echo " please use -f to force the minikube  cluster creation."
+
+
     fi
```

Comment:
I think the current does the job and the message is clear in case the minikube profile exists. We don't need to show anything in case everything is OK.